packages/main/src/ToolbarButton.ts

```diff
@@ -175,24 +172,18 @@ class ToolbarButton extends ToolbarItem {
 		return true;
 	}
 
-	static get toolbarTemplate() {
-		return ToolbarButtonTemplate;
-	}
-
-	static get toolbarPopoverTemplate() {
-		return ToolbarPopoverButtonTemplate;
-	}
-
 	onClick(e: Event) {
 		e.stopImmediatePropagation();
 		const prevented = !this.fireDecoratorEvent("click", { targetRef: e.target as HTMLElement });
 		if (!prevented && !this.preventOverflowClosing) {
 			this.fireDecoratorEvent("close-overflow");
 		}
 	}
-}
 
-registerToolbarItem(ToolbarButton);
+	get class() {
```

Comment:
Let's refactor the classes getter to return ClassMap:

**Parent**

```ts
	get classes() {
		return {
		    "ui5-tb-popover-item": this._isOverflowed,
		    "ui5-tb-item": true
		};
	}
```


**Child**
```ts
	/**
	 * @override
	 */
	get classes() {
		return {
		    ...super.classes,
		    "ui5-tb-button": true
		};
	}
```